cmd/server/main.go

```diff
@@ -40,6 +41,6 @@ func main() {
 
 	ctx := context.Background()
 	if err := server.Run(ctx, "tcp", cfg); err != nil {
-		log.Fatalf("got error starting server: %v", err)
+		os.Exit(1)
```

Comment:
Is there any reason changed this? I think Fatal is still appropriate for server errors.

Reply:
OK, will revert.